examples/internal_wave.jl

```diff
@@ -142,6 +142,10 @@ run!(simulation)
 
 w_timeseries = FieldTimeSeries(filename * ".jld2", "w")
 
+# And build the the ``x, y`` grid for plotting purposes.
+
+x, y, z = nodes(w_timeseries)
+
 #-
```

Comment:
What's this do?

Reply:
Breaks the cell in literate. Shows the output and then starts new code block.